src/test/java/PageFactoryTest.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.EmailsFactoryPage;
+import pages.LoginFactoryPage;
+
+public class PageFactoryTest extends TestBase {
+    public static String siteURL = "https://mail.ru/";
+    public String userName = "seleniumtests10";
+    public String password = "060788avavav";
+    public String domain = "mail.ru";
+
+    public PageFactoryTest() {
+        super(siteURL);
+    }
+
+    @Test
+    public void login() {
+        LoginFactoryPage loginFactoryPage = PageFactory.initElements(driver, LoginFactoryPage.class);
+        loginFactoryPage.login(userName, password, domain);
+        waitForAllElements();
```

Comment:
Remove this method, if you need explicit waiter, it should be placed in the end of login(...) method